GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -761,6 +789,40 @@ - (BOOL)application:(GULApplication *)application
 
 #endif  // TARGET_OS_IOS
 
+#pragma mark - [Donor Methods] UISceneDelegate URL handler
+
+#if UISCENE_SUPPORTED
+- (void)scene:(UIScene *)scene
+    openURLContexts:(NSSet<UIOpenURLContext *> *)URLContexts API_AVAILABLE(ios(13.0), tvos(13.0)) {
+  if (@available(iOS 13.0, tvOS 13.0, *)) {
+    SEL methodSelector = @selector(scene:openURLContexts:);
+    // Call the real implementation if the real App Delegate has any.
+    NSValue *openURLContextsIMPPointer =
+        [GULAppDelegateSwizzler originalImplementationForSelector:methodSelector object:self];
+    GULOpenURLContextsIMP openURLContextsIMP = [openURLContextsIMPPointer pointerValue];
+
+    // This is needed for the library to be warning free on iOS versions < 9.
```

Comment:
The comment does not look up to date. I guess, `pragma clang diagnostic ignored "-Wunguarded-availability"` may not be needed here at all because of `#if UISCENE_SUPPORTED`

Reply:
Yeah, you're right.